content/cpp/concepts/math-functions/terms/atan2/atan2.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'atan2()'
+Description: 'Returns the inverse tangent of coordinates in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`atan2()`** function returns the inverse tangent of a coordinate (y/x), where y is the y-coordinate value and x is the x-coordinate value.
+
+## Syntax
+
+```pseudo
+atan2(y, x)
+```
+
+The `y`, `x` parameter can be one of the following [data types](https://www.codecademy.com/resources/docs/java/data-types)
```

Comment:
This needs to mention the data types.

Reply:
@shlok-2003 You should list what data types x and y can be. It says "the following data types" but does not list the data types following the sentence.

Reply:
I have added the data type list in the file